.gitignore

```diff
@@ -1,13 +1,14 @@
-node_modules/
-bin/
```

Comment:
Are we no longer ignoring the bin folder?

Reply:
yes, this was ignored due to the fhir-validation binary being downloaded here, this is now donwloaded under models/dist and ignored uner the **/dist/ rule